stubs/pywin32/isapi/threaded_extension.pyi

```diff
@@ -1,26 +1,28 @@
 import threading
 from _typeshed import Incomplete
+from collections.abc import Callable
+from typing import Final, Literal
 
 import isapi.simple
 
-ISAPI_REQUEST: int
-ISAPI_SHUTDOWN: int
+ISAPI_REQUEST: Final = 1
+ISAPI_SHUTDOWN: Final = 2
 
 class WorkerThread(threading.Thread):
     running: bool
-    io_req_port: Incomplete
+    io_req_port: int
     extension: Incomplete
-    def __init__(self, extension, io_req_port) -> None: ...
+    def __init__(self, extension, io_req_port: int) -> None: ...
     def call_handler(self, cblock) -> None: ...
 
 class ThreadPoolExtension(isapi.simple.SimpleExtension):
     max_workers: int
     worker_shutdown_wait: int
-    workers: Incomplete
-    dispatch_map: Incomplete
-    io_req_port: Incomplete
+    workers: list[WorkerThread]
+    dispatch_map: dict[int, Callable[..., None]]
+    io_req_port: int
     def GetExtensionVersion(self, vi) -> None: ...
-    def HttpExtensionProc(self, control_block): ...
+    def HttpExtensionProc(self, control_block) -> Literal[3]: ...
```

Comment:
Given this is a base class and meant to be overwritten, I wouldn't restrict the return type of subclasses to a single status code.
```suggestion
    def HttpExtensionProc(self, control_block) -> int: ...
```